content/kit/pi/update-file.md

```diff
@@ -0,0 +1,27 @@
+---
+title: The update file
+---
+
+The update file is the simplest way to update your Pi with new software from us.  
```

Comment:
Where do they get such a file from?

Reply:
In PR description:

>This PR doesnt contain the page to download files, that'll come later

Although I'm thinking it'd be better to create a download page now, even if it just says "no updates available" for the time being.

Reply:
Creating this page is next on my list, however will be done outside this branch

Reply:
I'd rather we didn't merge this until it's clear to readers where the updates will come from, even that's only a placeholder for the moment.

Reply:
This is handled in #97

Reply:
#97 looks good; can we link to those pages in this PR?